respeaker_ros/README.md

```diff
@@ -69,6 +69,10 @@ A ROS Package for Respeaker Mic Array
     rostopic echo /is_speeching        # Result of VAD
     rostopic echo /audio               # Raw audio
     rostopic echo /speech_audio        # Audio data while speeching
+    rostopic echo /speech_to_text      # Voice recognition
+    # Voice recognition result for Japanese
+    rostopic echo --filter "print(m.transcript[0])" /speech_to_text
+    rostopic echo /speech_to_text | ascii2uni -a U -q
```

Comment:
`rostopic echo --filter "print('transcript: [%s]\n---'%(', '.join(map(str, m.transcript))))" /speech_to_text` is good?

Reply:
It's ok to use it. I'll add.